kolibri/plugins/learn/assets/src/views/LearningActivityIcon.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+
+  <KIcon :icon="icon" />
+
+</template>
+
+
+<script>
+
+  import { LearningActivityKinds } from 'kolibri.coreVue.vuex.constants';
+
+  const LearningActivityKindToIconMap = {
+    [LearningActivityKinds.CREATE]: 'create',
+    [LearningActivityKinds.LISTEN]: 'listen',
+    [LearningActivityKinds.REFLECT]: 'reflect',
+    [LearningActivityKinds.PRACTICE]: 'practice',
+    [LearningActivityKinds.READ]: 'read',
+    [LearningActivityKinds.WATCH]: 'watch',
+    [LearningActivityKinds.EXPLORE]: 'interact',
```

Comment:
In KDS, `explore` icons are called `interactSolid` and `interactShaded`. For details see https://learningequality.slack.com/archives/C023DNC5GCX/p1623711371049800